pages/utils.ts

```diff
@@ -0,0 +1,7 @@
+export const sanitizeURL = (url: string) => {
+    if (url.startsWith("http")) {
+        return url;
+    } else {
+        return `https://${url}`;
```

Comment:
This is needed since as per https://vercel.com/docs/concepts/projects/environment-variables/system-environment-variables, VERCEL_URL will not contains https protocol. Also, local development will be http